samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages.properties

```diff
@@ -7,7 +7,10 @@ add_q=Add Question:
 addr=Addresses:
 admin=Admin
 done=done
-ass_preview=Assessment Preview - This is an example student view of this assessment                                                            
+exit_preview=Exit Preview
+ass_preview_1=Assessment Preview - This preview provides an approximate view of the layout of the assessment for students.
+ass_preview_2= The preview will <strong>not</strong> allow you to save or submit answers, display the timer if there is a time limit, or access any feedback.
+ass_preview_3= The preview also does <strong>not</strong> take into account students' ability (or lack thereof) to view images or other media embedded in questions.
```

Comment:
You don't actually need to put these strings into 3 properties. You can do multiline properties in Java, just use a backslash at the end of each line, just for readability. That way the whole thing is loaded up into one element during the faces render. So, I'd just update ass_preview with whole suggested text, with those backslashes on the end of the lines.

Reply:
Hello, Adrian;
@adrianfish 
Thank again for the suggestions. 
Now there's a cleaner commit.

Greetings.